README.md

```diff
@@ -2,76 +2,77 @@
+  make rendering email easy.
 
 See the [docs] for the most up to date information.
 
-We designed Bamboo to be simple and powerful. If you run into _anything_ that is
-less than exceptional, or you just need some help, please open an issue.
-
-[docs]: https://hexdocs.pm/bamboo/readme.html
-
-## Adapters
+We designed Bamboo to be simple and powerful. If you run into _anything_ that
+is less than exceptional, or you just need some help, please open an issue.
 
-The Bamboo.MandrillAdapter and Bamboo.SendGridAdapter **are being used in production**
-and have had no issues. It's also pretty simple to [create your own adapter]. Feel free
-to open an issue or a PR if you'd like to add a new adapter to the list.
+**Do you like to learn by watching?**
 
-- `Bamboo.ConfigAdapter` - See [BinaryNoggin/bamboo_config_adapter](https://github.com/BinaryNoggin/bamboo_config_adapter) declare config at runtime.
-- `Bamboo.MailgunAdapter` - Ships with Bamboo. Thanks to [@princemaple].
-- `Bamboo.MailjetAdapter` - See [moxide/bamboo_mailjet](https://github.com/moxide/bamboo_mailjet).
-- `Bamboo.MandrillAdapter` - Ships with Bamboo.
-- `Bamboo.SendGridAdapter` - Ships with Bamboo.
-- `Bamboo.SMTPAdapter` - See [fewlinesco/bamboo_smtp](https://github.com/fewlinesco/bamboo_smtp).
-- `Bamboo.SparkPostAdapter` - See [andrewtimberlake/bamboo_sparkpost](https://github.com/andrewtimberlake/bamboo_sparkpost).
-- `Bamboo.PostmarkAdapter` - See [pablo-co/bamboo_postmark](https://github.com/pablo-co/bamboo_postmark).
-- `Bamboo.SendcloudAdapter` - See [linjunpop/bamboo_sendcloud](https://github.com/linjunpop/bamboo_sendcloud).
-- `Bamboo.SesAdapter` - See [kalys/bamboo_ses](https://github.com/kalys/bamboo_ses).
-- `Bamboo.FallbackAdapter` - See [fuelen/bamboo_fallback](https://github.com/fuelen/bamboo_fallback). Allows using multiple adapters.
-- `Bamboo.LocalAdapter` - Ships with Bamboo. Stores email in memory. Great for local development.
-- `Bamboo.TestAdapter` - Ships with Bamboo. Use in your test environment.
+In addition to this guide, check out the [free Bamboo screencast from
+DailyDrip]. It is a wonderful introduction to sending and testing emails with
+Bamboo. It also covers some of the ways that Bamboo helps catch errors, how
+some of the internals work, and how to format recipients with the
+[`Bamboo.Formatter`] protocol.
 
-[@princemaple]: https://github.com/princemaple
+## Installation
 
-To switch adapters, change the config for your mailer:
+To install Bamboo, add it to your list of dependencies in `mix.exs`. 
 
 ```elixir
-# In your config file
-config :my_app, MyApp.Mailer,
-  adapter: Bamboo.LocalAdapter
+def deps do
+  [{:bamboo, "~> 1.2"}]
+end
 ```
 
-[bamboo]: http://github.com/thoughtbot/bamboo
-[create your own adapter]: https://hexdocs.pm/bamboo/Bamboo.Adapter.html
-
-## Installation
-
-1.  Add bamboo to your list of dependencies in `mix.exs`:
+You may may also use the latest code available from master instead of a
```

Comment:
Duplicate `may`.

Reply:
Done and done! Thanks for the typo catch.